cores/rp2040/RP2040Support.h

```diff
@@ -314,6 +314,14 @@ class RP2040 {
         reboot();
     }
 
+    void wdt_begin(uint32_t delay_ms) {
+        watchdog_enable(delay_ms, 1);
+    }
+
+    void wdt_reset() {
+        watchdog_reboot();
```

Comment:
Err, don't you mean `watchdog_update()`?  This call should also fail because it actually resets the chip (see the `RP2040::reboot()` method in this file) and needs add'l params.

Reply:
Yeah, should be watchdog_update (https://raspberrypi.github.io/pico-sdk-doxygen/group__hardware__watchdog.html#ga329b748919954f8b48c58049115a5c54), to reload the counter with the specified time, not reset it.